plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -223,9 +223,18 @@ function exportSubmissions($submissionIds, $context, $user) {
 			$submission = $submissionDao->getById($submissionId, $context->getId());
 			if ($submission) $submissions[] = $submission;
 		}
-		$submissionXml = $exportFilter->execute($submissions);
-		if ($submissionXml) $xml = $submissionXml->saveXml();
-		else fatalError('Could not convert submissions.');
+		libxml_use_internal_errors(true);
+		$submissionXml = $exportFilter->execute($submissions, true);
+		$xml = $submissionXml->saveXml();
+		$errors = array_filter(libxml_get_errors(), create_function('$a', 'return $a->level == LIBXML_ERR_ERROR ||  $a->level == LIBXML_ERR_FATAL;'));
+		if (!empty($errors)) {
+			$charset = Config::getVar('i18n', 'client_charset');
+			header('Content-type: text/html; charset=' . $charset);
+			echo '<html><body>';
+			$this->displayXMLValidationErrors($errors, $xml);
+			fatalError(__('plugins.importexport.common.error.validation'));
+			echo '</body></html>';
```

Comment:
I think `fatalError` will stop execution, right? If so, the `echo` line should be above the `fatalError()` call.